src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,52 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoTicket {
+
+    private final List<LottoNumbers> tickets = new ArrayList<>();
+
+    public LottoTicket(final int ticketSize,
+        final LottoNumberSelectPolicy lottoNumberSelectPolicy) {
```

Comment:
```suggestion
    public LottoTicket(final int ticketSize,
                       final LottoNumberSelectPolicy lottoNumberSelectPolicy) {
```
팀마다 컨벤션은 다르겠지만... 자바에선 일반적으로 파라미터 기준으로 정렬을 맞출거에요ㅎㅎㅎ
(자동 포멧팅으로 설정가능합니다!)